frontend/packages/metal3-plugin/src/components/dashboard/health-card.tsx

```diff
@@ -41,8 +48,23 @@ const getHostHealthState = (obj): HostHealthState => {
   }
 };
 
-export const HealthCard: React.FC<HealthCardProps> = ({ obj }) => {
+const filterAlerts = (alerts: Alert[]): Alert[] =>
+  alerts.filter((alert) => _.get(alert, 'labels.hwalert'));
```

Comment:
@jtomasek can you please let me know what kind of filtering is required here ?

Reply:
@honza